### PR TITLE
chore: adjust checking of revert-error

### PR DESCRIPTION
**Pull request by keroro520**
godwoken-tests CI fails due to https://github.com/nervosnetwork/godwoken-web3/pull/512 merged.

### CODE REVIEW

File: .github/workflows/godwoken-tests.yml [hunk 1]
```diff
@@ -13,3 +13,6 @@ jobs:
       extra_github_env: |
         GODWOKEN_TESTS_REPO=${{ github.repository }}
         GODWOKEN_TESTS_REF=${{ github.ref }}
+        MANUAL_BUILD_WEB3=true
```
Comment (RetricSu) [inline .github/workflows/godwoken-tests.yml]: when can we remove this? after kicker update with the new web3 version?

**Review by RetricSu — APPROVED** [review]

File: .github/workflows/godwoken-tests.yml (hunk 1 above)
Comment (keroro520) [reply .github/workflows/godwoken-tests.yml]: Yes..

Comment (RetricSu) [reply .github/workflows/godwoken-tests.yml]: if we merge this pr now, will it affect the interaction test of other CIs, such as alphanet / testnet?

Comment (keroro520) [reply .github/workflows/godwoken-tests.yml]: Oh, yes, CI will fail on alphanet / testnet. That is a problem.
What about changing to compatible assertion?

```
      expect(error.message + error.error.toString()).to.include(keyWord);
```

Comment (RetricSu) [reply .github/workflows/godwoken-tests.yml]: great, I think this is a better solution!

Comment (keroro520) [reply .github/workflows/godwoken-tests.yml]: Updated in ~~e69db8d~~ [1e3c033](https://github.com/nervosnetwork/godwoken-tests/commit/1e3c0331478a2bc6105f919630474230960e5847)
And drop the updates for godwoken-tests.

**Pull request merged**